redi-connect-front/src/pages/app/me/Mentoring.tsx

```diff
@@ -14,11 +14,9 @@ import * as Yup from 'yup'
 
 import { FormikValues, useFormik } from 'formik'
 import {
-  categories
+  categories as availableCategories
 } from '../../../config/config'
 
-const availableCategories = Object.assign({}, ...categories.map(category => ({ [category.id]: category.label })))
-
 export interface MentoringFormValues {
```

Comment:
I've seen once or twice that you declare an interface, and add an `export`. No need to export unless the interface will be used in another file :)

Reply:
not sure why i did that, thanks for info